### ISSUE TITLE
Alwasy load home directory when open ranger.

### ISSUE THREAD

**Issue by LinArcX** [ranger/ranger#1405]
#### Runtime Environment

- Operating system and version:  Arch
- Terminal emulator and version: Konsole(v18.08.3)
- Python version: Python 3.7.1
- Ranger version/commit: 
- Locale: 
~ » ranger --version
ranger version: ranger 1.9.2
Python version: 3.7.1 (default, Oct 22 2018, 10:41:28) [GCC 8.2.1 20180831]
Locale: en_US.UTF-8

#### Current Behavior
I want to save and restore tabs. so i set the set save_tabs_on_exit to true. but when i exit from ranger with F10, when i back again(for example assume i saved 3 locations), always load a new tab(home directory). if i exit again and enter, new tab added(in this step, 5 tabs, and this behavoir continue!)

#### Expected Behavior
Only load tabs that i want.

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```

```

**Comment by toonn** [comment]
It's weird. What you describe is actually the behavior I'd expected. If you run `ranger` you normally expect it to open the current directory or the directories passed as arguments. However, that only happens every other time for me.

Best compromise I can think of is to only open directories if there's no tabs for them yet. So if you didn't have a tab for `PWD` it'd open a new one but if you *did* it would just focus it or focus the first of the arguments and create tabs for any arguments that don't exist yet.

**Comment by LinArcX** [comment]
Any idea to fix this issue?

**Comment by toonn** [comment]
I'd experiment with `ranger --cmd=tab_close`.

**Comment by LinArcX** [comment]
Yes. I hope this issue will fix in the next release.

**Comment by toonn** [comment]
Just to be clear, we may decide to fix this in the opposite direction of what you want. It's not obvious that the behavior is undesirable.

**Comment by LinArcX** [comment]
Yes. i know. one idea to fix my issue is close last tab(that open automatically by ranger each time)before exit the ranger. but i have no idea to do this.
